source/events/events.rst

```diff
@@ -0,0 +1,295 @@
+Each row of the table provides information that characterises one event.
+The mandatory and optional columns of the table are listed below. In
+addition, a list of header keywords providing metadata is specified.
+Also here there are mandatory and optional keywords.
+The suggested extension name of the binary table is ``EVENTS``.
+
+
+Mandatory columns
+-----------------
+
+* ``EVENT_ID`` tform: ``1J``
+    * Event identification number at the DL3 level
+      (lower data levels could be different, see note below).
+* ``TIME`` tform: ``1D``, unit: s
+    * Time stamp of event in instrument specific MJD time reference. See the
+      header keywords ``MJDREFI`` and ``MJDREFF`` for the zero point of
+      the reference time.
+      See also the `OGIP event list`_ standard.
+* ``RA`` tform: ``1E``, unit: deg
+    * Reconstructed event Right Ascension (see :ref:`sky-coordinates-radec`).
+      See also `HFWG Recommendation R3`_ for the OGIP standard.
+* ``DEC`` tform: ``1E``, unit: deg
+    * Reconstructed event Declination (see :ref:`sky-coordinates-radec`).
+      See also `HFWG Recommendation R3`_ for the OGIP standard.
+* ``ENERGY`` tform: ``1E``, unit: TeV
+    * Reconstructed event energy.
+      See also the `OGIP event list`_ standard.
+* ``DETX`` tform: ``1E``, unit: deg
+    * Reconstructed event X-coordinate in detector system
+      (nominal system, see :ref:`sky-coordinates-fov`).
+      See also the `OGIP event list`_ standard.
+* ``DETY`` tform: ``1E``, unit: deg
+    * Reconstructed event Y-coordinate in detector system
+      (nominal system, see :ref:`sky-coordinates-fov`).
+      See also the `OGIP event list`_ standard.
+* ``EVENT_TYPE`` tform: ``32X``
+    * Event quality partition.
+
+      .. note::
+         It needs to be discussed whether this should be a mandatory column
+         or an optional column. The partition of events into quality classes
+         seems sufficiently fundamental so that all IACT should in principle
+         support this. For example event multiplicity could be translated
+         into quality partitions, and the IRFs computed accordingly.
+
+
+Optional columns
+----------------
+
+.. note::
+   None of the following columns is required to be part of an ``EVENTS``
+   extension. Any software using these columns should first check whether the
+   columns exist, and warn in case of their absence.
+
+* ``ALT`` tform: ``1E``, unit: deg
+    * Reconstructed altitude coordinate of event
+      (horizon system, see :ref:`sky-coordinates-altaz`)
+* ``AZ`` tform: ``1E``, unit: deg
+    * Reconstructed azimuth coordinate of event
+      (horizon system, see :ref:`sky-coordinates-altaz`)
+* ``THETA`` tform: ``1E``, unit: deg
+    * Reconstructed offset from the observation pointing position
+* ``PHI`` tform: ``1E``, unit: deg
+    * Reconstructed position angle from the observation pointing position
+      (position angles are counted counterclockwise from celestial North)
```

Comment:
PS: if you do want to add links to the general section in this PR you should rebase your branch against latest master ... I did a few updates a few days ago.